src/app/public/styles/ag-grid-styles.scss

```diff
@@ -87,32 +92,23 @@ ag-grid-angular.sky-ag-grid-editable {
 
   /* fine tune focus cell borders */
   .ag-row {
-    .ag-cell-focus.ag-cell-inline-editing {
-      margin-top: $row-border-width * -1;
-      @include sky-field-status(active)
+    .ag-cell-focus {
+      line-height: $row-height - ($row-border-width * 4);
     }
 
-    .ag-cell-focus.ag-cell-not-inline-editing {
-      border: none;
-      border-right: $sky-standard-border;
-      box-shadow: none;
-    }
-  }
-
-  .ag-row.ag-row-first {
     .ag-cell-focus.ag-cell-inline-editing {
-      margin-top: 0;
+      height: $row-height - 1;
+      border-radius: 0;
+      @include sky-field-status(active)
     }
   }
 
-  .ag-row.ag-row-last {
-    .ag-cell-focus.ag-cell-inline-editing {
-      height: $row-height - 1;
-    }
+  .ag-row.ag-row-first {
+    height: $row-height - 1 !important;
```

Comment:
Could this CSS selector be more specific so that `!important` would not be needed?